python/llm/src/ipex_llm/transformers/low_bit_linear.py

```diff
@@ -852,7 +838,7 @@ def use_esimd_kernel(self, x):
         if self.disable_fp16_opt:
             return False
         # esimd kernel can only be used for Arc and Flex
-        if gpu_type not in ["arc", "flex"]:
+        if gpu_type not in ["arc"]:
```

Comment:
why remove flex here?

Reply:
flex is now recognized as `"arc"`, no need to handle it any more